ballerina/Ballerina.toml

```diff
@@ -2,7 +2,7 @@
 distribution = "2201.7.0"
 org = "ballerinax"
 name = "googleapis.gmail"
-version = "3.5.0"
+version = "4.0.0"
```

Comment:
Any reason to bump the major version in here? didn't notice any breaking changes from user's POV

Reply:
My idea was we are releasing major versions with this effort. My mistake. We can take that decision if/when we have API changes. ;)